vetspire_interview/test/support/factory.ex

```diff
@@ -0,0 +1,10 @@
+defmodule VetspireInterview.Factory do
+  use ExMachina.Ecto, repo: VetspireInterview.Repo
```

Comment:
Nice, love the usage of ExMachina (though I prefer default cases be named as such and that arbitrary things like "chihuahua" in test default be called out in a test if used. I was wondering how the insert in the setup was passing with a comparison to "chihuahua" until I saw this.